spec/features/studies/view_study_properties_spec.rb

```diff
@@ -26,4 +26,43 @@
     click_link 'Study details'
     expect(page).to have_content('HuMFre approval number: 12345')
   end
+
+  context 'with data release strategy' do
+
+    it 'displays HuMFre approval number for Open(ENA) data release strategy' do
+      study.study_metadata.data_release_strategy = Study::DATA_RELEASE_STRATEGY_OPEN
+      study.study_metadata.hmdmc_approval_number = '12345'
+      study.study_metadata.save!
+
+      login_user(user)
+      visit study_path(study)
+      click_link 'Study details'
+      expect(page).to have_content('What is the data release strategy for this study?: open')
+      expect(page).to have_content('HuMFre approval number: 12345')
+    end
```

Comment:
There are some lines of code repeating in other tests. If we add more tests in future it would make sense refactoring it a bit, but for the moment seems ok.
I would do something like adding some before..end setup clauses, and maybe defining study and strategy with let(..)